setup.py

```diff
@@ -28,7 +31,8 @@
     packages=setuptools.find_packages(),
     install_requires=install_requires,
```

Comment:
I think it's worth adding a note here about precedence between `install_requires` and `extras_require`, since in the strict case, `setuptools.setup` receives both the inputs from `REQUIREMENTS.txt` and `REQUIREMENTS-STRICT.txt`.

Reply:
This in fact is the reason `.[strict]` does not work. :(